### PR TITLE
Android Project Filter Test

### PR DESCRIPTION
**Pull request by tiginamaria**

### CODE REVIEW

**Comment by nbirillo** [comment]
@tiginamaria Should I review it?

File: kotlin-analysis-gradle/src/test/kotlin/org/jetbrains/research/ml/kotlinAnalysis/gradle/ExtractGradleDependenciesTest.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        @JvmStatic
+        @Parameterized.Parameters(name = "{index}: ({0}, {1})")
+        fun getTestData() = getInAndOutArray(
+            ::ExtractGradleDependenciesTest,
+            inExtension = Extension.EMPTY, outExtension = Extension.TXT
+        )
+    }
+
+    @Test
+    fun testExtractRootGradleFileFromProject() {
```
Comment (nbirillo) [inline kotlin-analysis-gradle/src/test/kotlin/org/jetbrains/research/ml/kotlinAnalysis/gradle/ExtractGradleDependenciesTest.kt:34]: This function is almost the same as the testExtractRootGradleFileFromProject from AndroidProjectTaggingTest. Could you create a class or a common function for them?

**Review by nbirillo — CHANGES_REQUESTED** [review]
Great! I have only one comment

File: kotlin-analysis-gradle/src/test/kotlin/org/jetbrains/research/ml/kotlinAnalysis/gradle/ExtractGradleDependenciesTest.kt (hunk 1 above)
Comment (tiginamaria) [reply kotlin-analysis-gradle/src/test/kotlin/org/jetbrains/research/ml/kotlinAnalysis/gradle/ExtractGradleDependenciesTest.kt:34]: Have just fix it in my next PR with modules extraction [I'll create it tonight].

Comment (nbirillo) [reply kotlin-analysis-gradle/src/test/kotlin/org/jetbrains/research/ml/kotlinAnalysis/gradle/ExtractGradleDependenciesTest.kt:34]: But it is a bad practice to merge code duplicates into the main branch...

**Review by nbirillo — APPROVED** [review]
Great! Thank you!

**Pull request merged**